### PR TITLE
fix(ui): fixed comments display

### PR DESCRIPTION
**Pull request by margdsq**
### Fix of the display of the comments 
In this PR, we just made a few alterations to the style for the proper display of comments.

### CODE REVIEW

**Review by Lil-Noe — APPROVED** [review]
Than you for this ! Looks good to me.

**Review by EduardoNeville — COMMENTED** [review]
no issues here

File: package.json [hunk 1]
```diff
@@ -42,6 +42,24 @@
     "expo-system-ui": "~4.0.2",
     "expo-web-browser": "~14.0.0",
     "@testing-library/react-native": "^12.8.1",
+
```
Comment (EduardoNeville) [inline package.json:45]: leaving spaces between packages is not good practice

File: package.json [hunk 2]
```diff
@@ -42,6 +42,24 @@
...
+    "expo-file-system": "~18.0.3",
+    "expo-font": "~13.0.1",
+    "expo-image-picker": "~16.0.1",
+    "expo-linking": "~7.0.2",
+    "expo-router": "~4.0.5",
+    "expo-splash-screen": "~0.29.9",
+    "expo-status-bar": "~2.0.0",
+    "expo-system-ui": "~4.0.2",
+    "expo-web-browser": "~14.0.1",
+
```
Comment (EduardoNeville) [inline package.json:62]: same here

File: package.json [hunk 3]
```diff
@@ -72,7 +90,9 @@
     "@types/react-native": "^0.73.0",
     "firebase-mock": "^2.2.4",
     "jest": "^29.7.0",
-    "jest-expo": "~52.0.0",
+
```
Comment (EduardoNeville) [inline package.json:93]: same here

File: package.json [hunk 4]
```diff
@@ -72,7 +90,9 @@
...
+    "jest-expo": "~52.0.1",
+
```
Comment (EduardoNeville) [inline package.json:95]: same here

File: app.json [hunk 5]
```diff
@@ -32,6 +32,7 @@
     "plugins": [
       "@react-native-firebase/app",
       "expo-router",
+
```
Comment (EduardoNeville) [inline app.json:35]: remove the space

**Pull request merged**